GVRf/Framework/framework/src/main/java/org/gearvrf/GVRActivity.java

```diff
@@ -139,6 +139,19 @@ protected void onCreate(Bundle savedInstanceState) {
         setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
 
         mRenderableViewGroup = (ViewGroup) findViewById(android.R.id.content).getRootView();
+        mDockEventReceiver = new DockEventReceiver(this,
```

Comment:
All the changes to GVRActivity should be reverted. This resurrects an older version of it.